components/cop_codegen/src/rpn_function.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+#[derive(FromDeriveInput, Debug)]
+#[darling(attributes(rpn_function))]
+pub struct RpnFunctionOpts {
+    ident: syn::Ident,
+    generics: syn::Generics,
+    #[darling(default)]
```

Comment:
I suggest removing the default, requiring explicit `#[rpn_function(args = 0)]`, making the reader easier to understand.

Reply:
Nice suggestion!